telluric/georaster.py

```diff
@@ -644,6 +664,18 @@ def save(self, filename, tags=None, **kwargs):
 
         """
 
+        folder = os.path.abspath(os.path.join(filename, os.pardir))
+        os.makedirs(folder, exist_ok=True)
+
+        if (
+            (self._image is None and self._filename is not None) and
+            (tags is None and not kwargs)
+        ):
+            shutil.copy(self._filename, filename)
```

Comment:
Could we add a comment here as a reminder that this is `rasterio.shutil`?

Reply:
Done.